plugins/tls_cert

```diff
@@ -3,15 +3,13 @@
 use strict;
 use warnings;
 
-# Very basic script to create TLS certificates for qpsmtpd
+# create TLS certificates for qpsmtpd
 use File::Temp qw/ tempfile tempdir /;
 use Getopt::Long;
+use Sys::Hostname;
```

Comment:
I just noticed the conditional.  I guess it doesn't use gethostbyname() on linux/unix :)